skeleton-tests/test_addon_theme.py

```diff
@@ -47,6 +47,15 @@ def test_addon_theme(tmpdir, capsys, config):
         os.path.join(tmpdir.strpath, config.package_name),
     )
 
+    subprocess.call(
```

Comment:
It would be better, if we integrate the isort call into the bobtemplate hook so that the test will also be ok after the user generates the package. This fix here only makes our test not fail.

Reply:
could you integrate it into base.py and call it in the post_render https://github.com/plone/bobtemplates.plone/blob/5fe1c59211633078dc5e795adad7781349e7e4be/bobtemplates/plone/addon.py#L138
We could also run this in other sub-templates. Because we never know what the packages names are and need to fix the isort after creating code in general. So all sub-templates need to call this in the post renderer hook.